packages/core/src/common/config-manager.ts

```diff
@@ -0,0 +1,182 @@
+import ts from 'typescript';
```

Comment:
Formatting and editor concerns seem like they're specific to the language server and wouldn't come up in other contexts like CLI, right? If so, this module probably doesn't belong under `common`.